src/Model/ManagerInterface.php

```diff
@@ -16,48 +16,8 @@
 /**
  * @author Sylvain Deloux <sylvain.deloux@ekino.com>
  */
-interface ManagerInterface
+interface ManagerInterface extends CommandManagerInterface, QueryManagerInterface
```

Comment:
Maybe we should deprecate this class in general?

All remaining methods expose internal functionality, which is not a good idea.

Reply:
I think so too, that would lead to not needing the traits anymore, people would implement several interfaces in several classes instead of just one. OK for the traits if you feel this is kind of a compulsory step towards that